mlflow/tracking/_model_registry/client.py

```diff
@@ -96,7 +96,7 @@ def get_latest_versions(self, name, stages=None):
         :param name: Name of the registered model to update.
         :param stages: List of desired stages. If input list is None, return latest versions for
                        for 'Staging' and 'Production' stages.
-        :return: List of `:py:class:`mlflow.entities.model_registry.ModelVersionDetailed` objects.
```

Comment:
That is good catch! Thank you.

Reply:
Some files still contain `ModelVersionDetailed`. I will replace them with `ModelVersion`.

https://github.com/mlflow/mlflow/search?q=ModelVersionDetailed&unscoped_q=ModelVersionDetailed

Reply:
Can I remove this section?

https://github.com/mlflow/mlflow/blame/eea0843e7bdcbcba59cd6d3cfb84de565f40071b/docs/source/rest-api.rst#L1937

Reply:
Don't worry about the documentation on `REST APIs` page. That will be autogenerated as a part of the release process.